analyzers/src/SonarAnalyzer.Common/Extensions/ISymbolExtensions.Roslyn.cs

```diff
@@ -0,0 +1,82 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.CodeDom.Compiler;
+
+namespace Microsoft.CodeAnalysis.Shared.Extensions;
+
+[GeneratedCode("Copied from Roslyn", "ca66296efa86bd8078508fe7b38b91b415364f78")]
```

Comment:
Note to myself:
Needs to be changed to `[ExcludeFromCodeCoverage]`
https://sonarsource.slack.com/archives/C06S7H06Z4Y/p1713354144963039?thread_ts=1713264344.421699&cid=C06S7H06Z4Y